assets/js/order-script.js

```diff
@@ -121,5 +122,72 @@ jQuery(document).ready(function($) {
 		
 	});
 
+	// save document
+	$( ".wcpdf-data-fields .wpo-wcpdf-save" ).on("click", function() {
```

Comment:
there's a lot of code duplication here with the regenerate code, I suggest to merge the two and apply exceptions (differences) where necessary

Reply:
Because of the data loaded after saving.

Reply:
was this comment supposed to be to another thread?

Reply:
Yes, sorry!